art_show/templates/emails/art_show/application.html

```diff
@@ -6,20 +6,6 @@
 this coming {{ event_dates() }}. Below is a copy of your application. You may
 <a href="{{ c.URL_BASE }}/art_show_applications/edit?id={{ app.id }}">update it here</a>.
 
-<br/><br/>Should your application be approved, you will be required to pay for your space to complete your reservation.
```

Comment:
This text was just moved.

Reply:
It's different text. Agent codes would still be a feature regardless of event, but the text provided by MFF is MFF-specific (particularly the date, but also arguably the policy it invokes).

Reply:
... to clarify, in this PR the text just literally moves down a few lines because I thought that looked better.